code/infrastructure-api/src/server.js

```diff
@@ -4,13 +4,15 @@ import bodyParser from 'body-parser';
 import logger from 'winston';
 import config from './config/config';
 import routes from './config/routes';
+import verifyToken from './auth/authMiddleware';
 
 logger.level = config.get('logLevel');
 logger.remove(logger.transports.Console);
 logger.add(logger.transports.Console, { timestamp: true, colorize: true });
 
 const app = express();
 app.use(bodyParser.json());
+app.all('*', verifyToken);
```

Comment:
Do you need the *? I thought that `app.use` just puts the function in as middleware.... I may be wrong.